mne_bids/tests/test_write.py

```diff
@@ -81,17 +81,18 @@
 @pytest.fixture(scope="session")
 def _bids_validate():
     """Fixture to run BIDS validator."""
-    shell = False
-    bids_validator_exe = ['bids-validator', '--config.error=41',
-                          '--config.error=41']
+    vadlidator_args = ['--config.error=41']
+    exe = os.getenv('VALIDATOR_EXECUTABLE', 'bids-validator')
```

Comment:
do developers have to now define this environment variable? Can you add it to the contributing guide?

Reply:
No this variable is optional – as you can see, if it's not set, we'll just use the `bids-validator` command. Or did you mean something else?